android_jni/avifandroidjni/src/main/java/org/aomedia/avif/android/AvifDecoder.java

```diff
@@ -110,10 +110,10 @@ public static boolean decode(ByteBuffer encoded, int length, Bitmap bitmap) {
    * @param bitmap The decoded pixels will be copied into the bitmap.
    *     If the bitmap dimensions do not match the decoded image's dimensions,
    *               then the decoded image will be scaled to match the bitmap's dimensions.
-   * @param threads Number of threads to be used for the AVIF decode. Zero means use number of CPU
-   *     cores as the thread count. Negative values are invalid. When this value is > 0, it is
-   *     simply mapped to the maxThreads parameter in libavif. For more details, see the
-   *     documentation for maxThreads variable in avif.h.
+   * @param threads Number of threads to be used for the AVIF decode. Zero means use the library
+   *     determined optimal value as the thread count. Negative values mean use number of CPU cores
```

Comment:
Previously negative values were invalid. I guess you want to provide a way to specify "use the number of CPU cores", so you now use negative values for that purpose?

Reply:
yes that is correct.